x-pack/plugins/security_solution/server/lib/product_features_service/product_features_service.ts

```diff
@@ -159,6 +174,36 @@ export class ProductFeaturesService {
           return response.notFound();
         }
       }
+
+      // This control ensures the action privileges have been registered by the productFeature service,
+      // preventing full access (`*`) roles, such as superuser, from bypassing productFeature controls.
+      const authz = request.route.options.security?.authz;
+      if (isAuthzEnabled(authz)) {
+        const disabled = authz.requiredPrivileges.some((privilegeEntry) => {
+          if (typeof privilegeEntry === 'object') {
+            if (privilegeEntry.allRequired) {
+              if (privilegeEntry.allRequired.some(isApiPrivilegeSecurityAndDisabled)) {
```

Comment:
It should be used visa versa, `privilegeEntry.allRequired.every` and `privilegeEntry.anyRequired.some`

Reply:
I am finding _disabled_ actions, so I use the oposite operation. This approach allows me to check the privilege is a security solution action and is disabled, at the same time.
You can take a look at the test for examples